letsdrill.rb

```diff
@@ -1,24 +1,31 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  case integer
+  when 90..100
+    "A"
+  when 80..89
+    "B"
+  when 70..79
+    "C"
+  when 60..69
+    "D"
+  else
+    "F"
+  end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+  array.sort{|x,y| x.length <=> y.length}.shift
```

Comment:
You might consider using [min_by](https://ruby-doc.org/core-2.5.0/Enumerable.html#method-i-min_by).